### PR TITLE
script to scrape composers from classical era

### PR DESCRIPTION
**Pull request by ShilpaGhanashyamGore**
@sacdallago : i dont understand why  am I unable to commit after formating the code ?
I get this message 
$ git commit -m "code formatted" scrape_dbPedia.js
On branch develop
Your branch is up-to-date with 'origin/develop'.

### CODE REVIEW

**Comment by angelinrashmi2000** [comment]
Shilpa look into the formatting

**Comment by sacdallago** [comment]
@angelinrashmi2000 please be more specific and use [the review button in the "files changed" section](https://help.github.com/articles/about-pull-request-reviews/) to propose changes if they are necessary, or approve the merge :)

**UPDATE**: See below.

File: scrape_dbPedia.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
+const http = require('http');
+var express = require('express');
```
Comment (sacdallago) [inline scrape_dbPedia.js:2]: `var` vs `const` vs `let`. Are they variables, constants or scope-specific? Be consistent with the use of these!

File: scrape_dbPedia.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+  server.listen(port, hostname, () => {
+  console.log(`Server running at http://${hostname}:${port}/`);
+
+  var elemNum = 0;
+
+  var newObj = {
+    table: []
+  };
+  fs.readFile('myjsonfile2.json', 'utf-8', function(err, array) {
```
Comment (sacdallago) [inline scrape_dbPedia.js:29]: [Here](https://github.com/MusicConnectionMachine/StructuredData/pull/12/files#diff-815b15a3748d5c262f4563790a730a0dR15) you use ES2015/ES6 syntax to write a function callback (`(param) ==> { //doSomething }`), while in this line you use ES5 (`function(param){ //doSomething }`). Stick to one standard, it's more readable :)

File: scrape_dbPedia.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    var json = JSON.stringify(newObj);
+    fs.writeFile('myjsonfile2.json', json, 'utf8');
+  });
+
+  function getResults(queryURL,callback) {
+
+	  var name = 'done'
+
+	  request(queryURL,function(error , response , html){
+								if(html){
```
Comment (sacdallago) [inline scrape_dbPedia.js:41]: This seems too much indented?

**Review by sacdallago — CHANGES_REQUESTED** [review]

**Comment by sacdallago** [comment]
@ShilpaGhanashyamGore you can't commit on develop, nor you should. See "Feature Branch Workflow" [here](https://www.atlassian.com/git/tutorials/comparing-workflows)

**Comment by ShilpaGhanashyamGore** [comment]
@sacdallago :Thank you for the review comments ! I shall incorporate them

**Comment by TimHenkelmann** [comment]
This was not a valid branch so we decided to close this PR.

**Pull request closed without merging**